config.xml

```diff
@@ -13,7 +13,13 @@
   <projectNamingStrategy class="jenkins.model.ProjectNamingStrategy$DefaultProjectNamingStrategy"/>
   <workspaceDir>${ITEM_ROOTDIR}/workspace</workspaceDir>
   <buildsDir>${ITEM_ROOTDIR}/builds</buildsDir>
-  <jdks/>
+  <jdks>
+    <jdk>
+      <name>Java 7</name>
+      <home>/Library/Java/JavaVirtualMachines/jdk1.7.0_51.jdk/Contents/Home</home>
```

Comment:
that is expected. i have been focusing on implementing the desired functionality before generalizing the configuration for other servers. i can create a ticket to configure jenkins on one of our virtual machines whenever you say so.